runtime/src/lib.rs

```diff
@@ -229,10 +229,100 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
         );
     }
 
+    let mut external_cmd_buffer =
```

Comment:
Can you refactor this into a helper function? Otherwise the main command handler function is getting pretty messy.